src/main/java/net/imagej/plugins/io/mesh/stl/BinarySTLFormat.java

```diff
@@ -94,34 +96,34 @@
 	private static void writeFacet(final ByteBuffer buffer,
 		final STLFacet facet)
 	{
-		writeVector(buffer, facet.normal);
-		writeVector(buffer, facet.vertex0);
-		writeVector(buffer, facet.vertex1);
-		writeVector(buffer, facet.vertex2);
+		writeVector(buffer, facet.getNormal());
```

Comment:
This is great, but better to do as two separate commits. The first one adds @tibuch's mesh classes, and the second one changes the code here accordingly?